firmware/user/user_main.c

```diff
@@ -171,8 +199,10 @@ void ICACHE_FLASH_ATTR user_init(void)
 #endif
 
     // Initialize LEDs
+#ifndef GDBSTUB_H
```

Comment:
Stick with `USE_ESP_GDB` for consistency

Reply:
Richard